### ISSUE TITLE
Changing the DNS doesn't work

### ISSUE THREAD

**Issue by dude365635** [Blobby-Boi/Blobwifi#3]
Just a simple DNS change breaks my wifi network....
It doesn't work to just change the DNS by the looks of it
But cool script I'm sure it will help other people just not me :(

**Comment by Blobby-Boi** [comment]
How does it break the Wi-Fi network? You might want to try changing the dns servers to other public ones.

**Comment by dude365635** [comment]
Hello,
So I downloaded the file, followed the instructions very carefully, and when I got to school I tested it and the wifi just would not work It connected but it wouldn't load a webpage! Any ideas? 
Thanks in advance!

**Comment by funkymokey** [comment]
I have this same issue, it worked just a tiny bit before it went down. I want to remove it.

**Comment by Blobby-Boi** [comment]
To remove it, go to chrome://os-settings and find the wifi. From there, change the DNS servers back.

**Comment by Sunniestar** [comment]
> How does it break the Wi-Fi network? You might want to try changing the dns servers to other public ones.

my school has a proxy that blocks everything aswell as content keeper

**Comment by killsecurly** [comment]
Simply using Google's DNS won't work.

**Comment by Blobby-Boi** [comment]
It does. It's not meant to bypass any filter extensions like cauDNS. This was meant to be a fast and easy way to bypass school Wi-Fi restrictions, which can be done with any public DNS.

**Comment by BridaNewb** [comment]
I don't want to open a new issue for a simple question. I'm not able to test this yet but maybe you can respond before I am able to anyway and save me time?

If I replace the network with the newly created .onc file, will I be able to go into that network to view the password? Right now, the Chromebook is locked down and I tried looking in prefs-internal and sync already. I found my other psks but not the school one. I can't expand it in settings. If I replace it with the .onc, will I be able to see it or view it in settings or in prefs-internal or chrome://sync-internals?

Thanks

**Comment by killsecurly** [comment]
> I don't want to open a new issue for a simple question. I'm not able to test this yet but maybe you can respond before I am able to anyway and save me time?
> 
> If I replace the network with the newly created .onc file, will I be able to go into that network to view the password? Right now, the Chromebook is locked down and I tried looking in prefs-internal and sync already. I found my other psks but not the school one. I can't expand it in settings. If I replace it with the .onc, will I be able to see it or view it in settings or in prefs-internal or chrome://sync-internals?
> 
> Thanks

No, you can't get the password. ONC simply forces the internal configuration to change to the selected DNS.

**Comment by BAaaakkka** [comment]
i did it but it simpy did nothing. i followed all the instructions but nothing changed. just to be clear this makes it so I can open blocked websites right?
